src/main/java/com/gary/backendv2/model/dto/request/TrustedPersonRequest.java

```diff
@@ -0,0 +1,30 @@
+package com.gary.backendv2.model.dto.request;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.lang.Nullable;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Null;
+
+@Getter
+@Setter
+public class TrustedPersonRequest {
+	@Email
+	@NotBlank
+	private String userEmail;
+
+	@NotBlank
+	private String firstName;
+
+	@NotBlank
+	private String lastName;
+
+	@Email
+	@Nullable
```

Comment:
czemu nullable?

nie jestem pewnien czy null albo pusty string przejdzie walidację emaila i moze być tak, że walidator nigdy nie przyjmie rządania z brakiem tego pola

Reply:
Z tego co pamiętam to u Adama było to zrobione jako opcjonalne.
Dam Unit test do tego